packages/extension-base/src/background/types.ts

```diff
@@ -23,11 +23,22 @@ type NullKeys<T> = { [K in keyof T]: IsNull<T, K> }[keyof T];
 
 export type SeedLengths = 12 | 24;
 
-export interface AccountJson {
+export interface AccountJson extends KeyringPair$Meta {
   address: string;
   genesisHash?: string | null;
   isExternal?: boolean;
   name?: string;
+  parentAddress?: string;
+  derivationPath?: string;
```

Comment:
Not convinced we want to store this at all - it could also contain a password `///<Password>` which mean that when exported, that is all in the clear.

Reply:
We can't include ///<Password> into derivation path for some reason. I am getting `Error: Re-constructed path "//path//password" does not match input` error from keyring when trying to do so

Reply:
That is weird. Will look into that, weirdly I did the same on the apps UI day or so ago and it was fine of account -> account derivation. Don't worry about it here, will play and try to track it down to not waste time.

Reply:
I thought, it was intended to be like that :). I'll remove derivationPath from metadata then.